### PR TITLE
add shor alg kubeflow pipeline example

### PR DESCRIPTION
**Pull request by zed546213**
This PR is to wrap quantum SHOR algorithm within a kubeflow pipeline.  The purpose is to allow factoring a large composite factor.  Shor's Algorithm, named after mathematician Peter Shor, is a quantum algorithm designed to efficiently factorize large composite numbers. It's one of the most famous and impactful algorithms in quantum computing, as it provides an exponential speedup over the best-known classical algorithms for factoring.

### CODE REVIEW

**Comment by rimolive** [comment]
@zed546213 Are you sure this PR is rebased?

**Comment by HumairAK** [comment]
it is not, @zed546213 can you rebase your pr?

**Comment by zed546213** [comment]
Sorry, i don't know how to rebase.

**Comment by zed546213** [comment]
@rimolive  @HumairAK  rebase has been completed.

File: shor_alg/shor_alg_pipeline.py
Comment (PaulinaPacyna) [inline shor_alg/shor_alg_pipeline.py:1]: consider moving to samples/contrib directory

**Pull request closed without merging**